community-operators/grafana-operator/3.6.0/grafana-operator.v3.6.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,221 @@
+                            fieldPath: metadata.namespace
+                      - name: POD_NAME
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.name
+                      - name: OPERATOR_NAME
+                        value: grafana-operator
+                    image: quay.io/integreatly/grafana-operator:v3.6.0
+                    imagePullPolicy: Always
+                    name: grafana-operator
+                    ports:
+                      - containerPort: 60000
+                        name: metrics
+                    resources: { }
+                serviceAccountName: grafana-operator
+      permissions:
+        - rules:
+            - apiGroups:
+                - ''
+              resources:
+                - pods
+                - services
+                - endpoints
+                - persistentvolumeclaims
+                - events
+                - configmaps
+                - secrets
+                - serviceaccounts
+              verbs:
+                - '*'
+            - apiGroups:
+                - apps
+              resources:
+                - deployments
+                - deployments/finalizers
+                - daemonsets
+                - replicasets
+                - statefulsets
+              verbs:
+                - '*'
+            - apiGroups:
+                - route.openshift.io
+              resources:
+                - routes
+                - routes/custom-host
+              verbs:
+                - '*'
+            - apiGroups:
+                - monitoring.coreos.com
+              resources:
+                - servicemonitors
+              verbs:
+                - get
+                - create
+            - apiGroups:
+                - extensions
+              resources:
+                - ingresses
+              verbs:
+                - '*'
+            - apiGroups:
+                - integreatly.org
+              resources:
+                - grafanas
+                - grafanadashboards
+                - grafanadatasources
+                - grafanas/finalizers
+                - grafanas/status
+                - grafanadashboards/status
+                - grafanadatasources/status
+              verbs:
+                - '*'
+          serviceAccountName: grafana-operator
+    strategy: deployment
+  installModes:
+    - supported: true
+      type: OwnNamespace
+    - supported: true
+      type: SingleNamespace
+    - supported: false
+      type: MultiNamespace
+    - supported: false
+      type: AllNamespaces
+  keywords:
+    - Grafana
+    - Metrics
+    - Monitoring
+  links:
+    - name: Documentation
+      url: https://github.com/integr8ly/grafana-operator/tree/v3.6.0/documentation
+    - name: Grafana
+      url: https://grafana.com
+  maintainers:
+    - email: integreatly-dev@redhat.com
+      name: Red Hat
+  maturity: alpha
+  provider:
+    name: Red Hat
+  replaces: 3.5.0
```

Comment:
The value for replaces has to be as name in csv file of that version. So like this `grafana-operator.v3.5.0`